data/templates/css/wikimedia.scss

```diff
@@ -1,5 +1,4 @@
 @import "project-setup";
-@import "compass";
```

Comment:
There is also `@include pie-clearfix;` from [compass](http://compass-style.org/reference/compass/utilities/general/clearfix/) but not exactly sure what it does.